.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+#### What does this PR do?
+
+[GPII-XXXX](https://issues.gpii.net/browse/GPII-XXXX)
```

Comment:
Usually ticket number (if any) is already in PR's title. Jira likes that.

Reply:
The idea here is to save clicks/searches to the reviewer.